src/main/resources/templates/appellant_appeal_welsh_template.html

```diff
@@ -512,8 +512,7 @@ <h1 class="heading-large">Appeal a benefit decision</h1>
     <dl class="govuk-check-your-answers">
         <div>
             <dt class="cya-question">Benefit name</dt>
-            <dd class="cya-answer">{{ PdfWrapper.sscsCaseData.appeal.benefitType.description }} ({{
-                PdfWrapper.sscsCaseData.appeal.benefitType.code }})
+            <dd class="cya-answer">{{ PdfWrapper.englishBenefitName }}</dd>
```

Comment:
definitely english benefit name?

Reply:
ha - well we already have 'welshBenefitName' so i was trying to be consistent